GLMakie/src/drawing_primitives.jl

```diff
@@ -266,13 +266,30 @@ function draw_atomic(screen::GLScreen, scene::Scene,
         glyphcollection = x[1]
         liftkeys = (:position, :rotation, :model, :space, :offset)
         args = getindex.(Ref(gl_attributes), liftkeys)
-        function collect_glyph_data(projview, transfunc, pos, rotation, model, space, offset)
-            gcollection = glyphcollection[]
-            res = Vec2f0(widths(pixelarea(scene)[]))
-            positions, offset, uv_offset_width, scale = preprojected_glyph_arrays(pos, gcollection, space, projview, res, offset, transfunc)
+
+        # TODO: This is a hack before we get better updating of plot objects and attributes going.
+        # Here we only update the glyphs when the glyphcollection changes, if it's a singular glyphcollection.
+        # The if statement will be compiled away depending on the parameter of Text.
+        # This means that updates of a text vector and a separate position vector will still not work if only the text
+        # vector is triggered, but basically all internal objects use the vector of tuples version, and that triggers
+        # both glyphcollection and position, so it still works
+        if glyphcollection[] isa Makie.GlyphCollection
```

Comment:
does this need the same fix in WGLMakie?

Reply:
Ah yeah always forget